tweets/views.py

```diff
@@ -16,17 +16,29 @@ def showTweet(request,id):
 
 def editTweet(request,id):
   tweet = get_object_or_404(Tweet, id=id, user_id=request.user.id)
+  if tweet.original_tweet is not None:
+   return HttpResponse("Retweets can't be retweeted", status=422)
```

Comment:
Wrong message

Reply:
this one is given by you above. where I suggested some other error messages